pypd/mixins.py

```diff
@@ -45,6 +46,13 @@ def __init__(self, api_key=None, base_url=None):
             from pypd import base_url
             self.base_url = base_url
 
+        if proxies:
```

Comment:
Based on the environment variable discussion above (I realized afterward) - if you explicitly pass a `None` to the requests function for the `proxies` variable, does it still respect the environment variables if set?